packages/core-api/src/app/App.tsx

```diff
@@ -17,19 +17,13 @@
 import React, { ComponentType, FC } from 'react';
```

Comment:
could perhaps even name this file `PrivateAppImpl.tsx` if you wanted to

Reply:
Yeah maybe, not this PR though imo